madmom/ml/nn/layers.py

```diff
@@ -344,6 +344,144 @@ def activate(self, data):
         return out
 
 
+class GRUCell(object):
+    """
+    Cell as used by GRU layers proposed in [1].
+
+    Parameters
+    ----------
+    weights_in_to_hidden_update : numpy array, shape ()
+        Weights.
+    weights_in_to_hidden_update : numpy array, shape ()
+        Weights.
+    bias : scalar or numpy array, shape ()
+        Bias.
+    activation_fn : numpy ufunc, optional
+        Activation function.
+
+    References
+    ----------
+    .. [1] Cho, Kyunghyun, et al,
+           "On the properties of neural machine translation: Encoder-decoder
+           approaches",
+           arXiv preprint arXiv:1409.1259 (2014).
```

Comment:
Would be nice to have this similarly formatted as all the other references (title first, complete names, link to arXiv)